examples/lifelong_learning/atcii/inference.py

```diff
@@ -31,12 +31,24 @@ def main():
     utd_parameters = Context.get_parameters("UTD_PARAMETERS", {})
     ut_saved_url = Context.get_parameters("UTD_SAVED_URL", "/tmp")
 
+    task_mining = {
+        "method": "TaskMiningByDataAttr",
+        "param": attribute
+    }
+
+    unseen_task_detect = {
+        "method": utd,
+        "param": utd_parameters
+    }
+
     ll_job = LifelongLearning(
```

Comment:
we still  name it job? what are other guys thoughts?

Reply:
Have discussed with other contributers that the job in CRD is named differently at different stages, training is the `job`, and inference will be a `service`. Please let me know if there's any better advice.

Also @llhuii @jaypume @TymonXie @JimmyYang20 @MooreZheng PTAL

Reply:
ok